src/qt/bitcoingui.cpp

```diff
@@ -707,6 +714,24 @@ void BitcoinGUI::openClicked()
     }
 }
 
+void BitcoinGUI::openWalletClicked()
+{
+    QString fileName = QFileDialog::getExistingDirectory(this, tr("Open Wallet"),
+                                                QString(),
+                                                QFileDialog::ShowDirsOnly
```

Comment:
Why only show directories and not symlinks? Wallets can be `.dat` files.

Reply:
Right.